tests/datadriven/__init__.py

```diff
@@ -194,6 +200,24 @@ def assertRaisesWith(self, exceptionType):
                 "exception of type {} not raised".format(
                     exceptionType.__name__))
 
+    def assertAlmostEqual(self, a, b, ndigits=7):
+        """
+        Assert that a and b are equal within ndigits digits
+        """
+        if round(a-b, ndigits) != 0:
```

Comment:
`round` only really makes sense when you are dealing with floating point numbers, and that's why round deals with decimal points.

But in any case, I got my inspiration for the implementation from the unittest spec... see the link above.